packages/kaoto-ui/cypress/e2e/00-iframe-test/about.cy.js

```diff
@@ -0,0 +1,32 @@
+describe('Settings: About', () => {
```

Comment:
Adapted from [packages/kaoto-ui/cypress/e2e/01-settings/about.cy.js](https://github.com/KaotoIO/kaoto-ui/blob/7c803defcea2b6ce95b615b806aba55e86c1f0dc/packages/kaoto-ui/cypress/e2e/01-settings/about.cy.js)

Reply:
The change is to replace `.get()` functions with `getIframeBody()` calls when applies. I'm wondering if this can be inferred at runtime, meaning that we will use the same function no matter if we're running the test against `Kaoto UI` or the sandbox.